PTGLgraphComputation/tools/plcc_cluster/settings_statistics.cfg

```diff
@@ -49,6 +49,8 @@ PLCC_OPTIONS="--cluster --outputdir $PLCC_OUTPUT_DIR"				# WARNING: make sure ou
 OUTPUT_COPY_DIR_ALL_NODES="/shares/modshare/vplg_all_nodes_output"		# The directory where the MPI version will copy the output generated on the nodes (in their tmp dir). Should be a permenent storage location reachable for all nodes (a mounted disk via NFS or another network FS)
 UPDATE_EXISTING_PDB_FILES="NO"                                      # whether PDB files which already exist in the database should be handled. if not, they will be skipped completely. useful for incremental updates.
 
+USE_BIOLOGICAL_ASSEMBLY_CIFFILE="YES"                               # whether to generate and use the biological assembly cif file
```

Comment:
As we need DSSP4 to create the DSSP files for the changed CIF files, we should leave that to "NO" for now.